modules/table/src/test/java/org/apache/ignite/internal/table/distributed/replicator/action/RequestTypeTest.java

```diff
@@ -58,4 +59,11 @@ void isWriteWorksAsExpected() {
             }
         }
     }
+
+    @Test
+    void testFromOrdinal() {
```

Comment:
Do we have similar tests for other enums that you updated? Did I miss them?

Reply:
No. There were no test classes for other enumerations, so I didn’t add them. Should I add them?

Reply:
yes, since you've added the serialization mechanism for them

Reply:
Added tests.